core/sequencer/server.go

```diff
@@ -327,6 +327,10 @@ func (s *Server) ApplyRevision(ctx context.Context, in *spb.ApplyRevisionRequest
 		return nil, err
 	}
 
+	// Verify that we are still master.
+	if err := ctx.Err(); err != nil {
```

Comment:
Would `SetLeavesAtRevision` not return if `ctx` is done?

Reply:
I hope so, I can't seem to actually find the code that would do that

Reply:
I guess it should be in grpc code that underpins the call?